core/server/worker/src/main/java/alluxio/worker/block/CacheRequestManager.java

```diff
@@ -67,12 +71,13 @@
    * @param blockWorker handler to the block worker
    * @param fsContext context
    */
-  public AsyncCacheRequestManager(ExecutorService service, BlockWorker blockWorker,
-      FileSystemContext fsContext) {
-    mAsyncCacheExecutor = service;
+  public CacheRequestManager(ExecutorService service, BlockWorker blockWorker,
+                             FileSystemContext fsContext) {
+    mCacheExecutor = service;
     mBlockWorker = blockWorker;
     mFsContext = fsContext;
-    mPendingRequests = new ConcurrentHashMap<>();
+    mPendingAsyncRequests = new ConcurrentHashMap<>();
```

Comment:
In terms of accounting, It shouldn't matter if it's async or sync. Why don't we just merge these two together?
And by merging these queues, we can also easily avoid async requests when there's already a sync request. And for the other direction, there's a race already with separate queues, so no change there.